CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Version 4.9.3
 
 _Unreleased_
 
+* Breaking: Encoding and decoding negative ints was broken in Swift and did not match protobuf spec.
```

Comment:
- [ ] Change "ints" to "`int32`s" (and back ticked appropriately)

Reply:
fixing and will update more on this.

Reply:
Updated and included more info